Homework3/Tests.cs

```diff
@@ -0,0 +1,44 @@
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using YandexMailClassLibrary;
+
+namespace Homework3
+{
+    public class Tests
+    {
+        private WebDriver _webDriver;
+        private YandexMailBoxPage _mailBoxPage;
+
+        [Test]
+        [TestCase("https://yandex.com/", "udod.udodovich@yandex.com", "cxzASDewq123567")]
+        public void LoginToYandexMailBox_UserEntersValidCredentials_LoginIsSuccessfullTest(string url, string username, string password)
+        {
+            _webDriver = new ChromeDriver();
+            _webDriver.Manage().Window.Maximize();
+            _webDriver.Navigate().GoToUrl(url);
+
+            YandexStartPage startPage = new YandexStartPage(_webDriver);
+            YandexMailHomePage mailHomePage = startPage.GoToMailHomePage();
+            _mailBoxPage = mailHomePage.Login(username, password);
+
+            Assert.Pass("Login test to Yandex has passed successfully!");
+        }
+
+        [Test]
+        public void LogoutFromYandex_UserPressesLogout_LogoutIsSuccessfullTest()
+        {
+            if (_mailBoxPage != null)
```

Comment:
I recommend you to write the full test here with log in operation. It will be as Pre-Condition for your test. But don't create tests which can be dependent on each other

Reply:
I've created a Private region with LoginToYandexMailBox() method that I use in each from two tests: 1) Login; 2) Login & Logout.